src/commonMain/kotlin/org/treeWare/model/operator/Difference.kt

```diff
@@ -277,11 +273,35 @@ private class DifferenceVisitor : AbstractLeaderManyModelVisitor<TraversalAction
         val updateField = updateStack.removeFirst() as MutableFieldModel
         if ((elementInclusion.inUpdate) || isKeyField(updateField)) { //Case: In or above an updated node
             val updateParent = updateStack.first() as MutableBaseEntityModel
+            if (updateField.elementType == ModelElementType.LIST_FIELD) copy(checkNotNull(newField), updateField)
             connectField(updateParent, updateField)
         }
 
         inclusionStack.first().addChildInclusions(elementInclusion)
     }
+
+
+    /**===== Uses the difference operator to compare associations =====**/
+    private fun associationsEqual(oldAssociation: AssociationModel, newAssociation: AssociationModel): Boolean {
+        val oldPathTree = oldAssociation.value
+        val newPathTree = newAssociation.value
+
+        val oldModel = MutableMainModel(mainFormat)
+        val oldRoot = oldModel.getOrNewRoot()
+        copy(oldPathTree, oldRoot)
+
+        val newModel = MutableMainModel(mainFormat)
+        val newRoot = newModel.getOrNewRoot()
+        copy(newPathTree, newRoot)
+
+        val differences = difference(oldModel, newModel)
+
+        val create = differences.createModel
+        val delete = differences.deleteModel
+        val update = differences.updateModel
+
+        return (create == null && delete == null && update == null)
```

Comment:
nit: add an `isDifferent()`  method to the `DifferenceModels` class to see if there are any differences